src/core/figures.js

```diff
@@ -21,7 +21,7 @@ const localizationStrings = {
     fig: "Figure ",
   },
   ja: {
-    fig: "図",
+    fig: "図 ",
```

Comment:
This is causing test failure, could you also revise the relevant test in `tests/spec/core/figures-spec.js:52:33`?

Reply:
fixed.